growsocial/common/methods/errorLogsMethods.js

```diff
@@ -1,10 +1,77 @@
 Meteor.methods({
-  addErrorLog: function(error) {
-    ErrorLogs.insert({
-      userId: Meteor.userId(), // null if not logged in
-      dateTime: new Date(),
-      tag: error.tag,
-      message: error.message
-    });
-  },
+	// check: function(object, type){
+	// 	switch(type){
+	// 		case "number":
+	// 		break;
+	// 		case "string":
+	// 		break;
+
+	// 	}
+	// },
+
+	addErrorLog: function(error) {
+
+	  	var err = {
+	  		userId: Meteor.userId(), // null if not logged in
+	  		dateTime: new Date()
+	  	};
+
+	  	// Tag (Type of error) (Max 30 characters)
+	  	if (typeof error.tag === 'string') {
```

Comment:
I'd rather use the check function, the package is already added. See meteor docs: http://docs.meteor.com/#/full/check_package